test/server/dummy-data.js

```diff
@@ -1,15 +1,16 @@
-module.exports = {
-  auth: {
+export let auth = {
     "patty.lastname@ttu.edu": "patty123",
     "john.doesnt@ttu.edu": "john234",
     "sally.does@ttu.edu": "324sally"
-  },
-  emailToUid: {
+  }
```

Comment:
Please fix indentation.  Also, this statement should end with a semicolon after the closing brace.